src/citrine/resources/material_template.py

```diff
@@ -115,3 +116,41 @@ class MaterialTemplateCollection(DataConceptsCollection[MaterialTemplate]):
     def get_type(cls) -> Type[MaterialTemplate]:
         """Return the resource type in the collection."""
         return MaterialTemplate
+
+    def get_specs(self, scope: str, id: str) -> dict:
+        """
+        Get all material specs associated with a material template.
+
+        The material template is specified by its scope and id.
+
+        :param scope: The scope used to locate the material template.
+        :param id: The unique id corresponding to `scope`.
+            The lookup will be most efficient if you use the Citrine ID (scope='id')
+            of the material template.
+        :return: A search result of material specs
+        """
+        path = self._get_path(ignore_dataset=True) + "/" + scope + "/" + id + "/material-specs"
+        return self.session.get_resource(path)
+
+    def get_runs(self,
```

Comment:
Comment above, considering that you've implemented two methods with the name `get_runs`.